metisp/pymetis/src/pymetis/classes/inputs/common.py

```diff
@@ -56,71 +63,89 @@ class MasterDarkInput(SinglePipelineInput):
     _title: str = "master dark"
     _tags: Pattern = re.compile(r"MASTER_DARK_(?P<detector>2RG|GEO|IFU)")
     _group: cpl.ui.Frame.FrameGroup = cpl.ui.Frame.FrameGroup.CALIB
+    _description: str = "Master dark frame for {det} detector data"
 
 
 class MasterFlatInput(SinglePipelineInput):
     _title: str = "master flat"
-    _tags: Pattern = re.compile(r"MASTER_IMG_FLAT_LAMP_(?P<detector>2RG|GEO|IFU)")
+    _tags: Pattern = re.compile(r"MASTER_IMG_FLAT_LAMP_(?P<band>LM|N)")
     _group: cpl.ui.Frame.FrameGroup = cpl.ui.Frame.FrameGroup.CALIB
+    _description: str = "Master flat frame for {det} data"
 
 
 class LinearityInput(SinglePipelineInput):
     _title: str = "linearity map"
     _tags: Pattern = re.compile(r"LINEARITY_(?P<detector>2RG|GEO|IFU)")
     _group: cpl.ui.Frame.FrameGroup = cpl.ui.Frame.FrameGroup.CALIB
+    _description: str = "Coefficients for the pixel non-linearity correction."
 
 
 class BadpixMapInput(SinglePipelineInput):
     _title: str = "bad pixel map"
     _tags: Pattern = re.compile(r"BADPIX_MAP_(?P<detector>2RG|GEO|IFU)")
     _group: cpl.ui.Frame.FrameGroup = cpl.ui.Frame.FrameGroup.CALIB
+    _description: str = "Bad pixel map. Also contains detector masks."
 
 
 class PersistenceMapInput(SinglePipelineInput):
     _title: str = "persistence map"
     _tags: Pattern = re.compile(r"PERSISTENCE_MAP")
     _group: cpl.ui.Frame.FrameGroup = cpl.ui.Frame.FrameGroup.CALIB
-    _required: bool = False     # Persistence maps are usually optional (but this can be overridden)
+    _description: str = "Persistence map"
+    _required = False # By default, persistence maps are optional
 
 
 class GainMapInput(SinglePipelineInput):
     _title: str = "gain map"
     _tags: Pattern = re.compile(r"GAIN_MAP_(?P<detector>2RG|GEO|IFU)")
     _group: cpl.ui.Frame.FrameGroup = cpl.ui.Frame.FrameGroup.CALIB
+    _description: str = "Gain map."
 
 
 class DistortionTableInput(SinglePipelineInput):
     _title: str = "distortion table"
     _tags: Pattern = re.compile(r"IFU_DISTORTION_TABLE")
     _group: cpl.ui.Frame.FrameGroup = cpl.ui.Frame.FrameGroup.CALIB
+    _description: str = "Table of distortion coefficients for an IFU data set"
 
 
 class WavecalInput(SinglePipelineInput):
     _title: str = "wavelength calibration"
     _tags: Pattern = re.compile(r"IFU_WAVECAL")
     _group: cpl.ui.Frame.FrameGroup = cpl.ui.Frame.FrameGroup.CALIB
+    _description: str = "Image with wavelength at each pixel"
 
 
 class PinholeTableInput(SinglePipelineInput):
     _title: str = "pinhole table"
     _tags: Pattern = re.compile(r"PINHOLE_TABLE")
     _group: cpl.ui.Frame.FrameGroup = cpl.ui.Frame.FrameGroup.CALIB
+    _description: str = "Table of pinhole locations"
 
 
-class FluxTableInput(SinglePipelineInput):
+class FluxstdCatalogInput(SinglePipelineInput):
     _title: str = "flux standard star catalogue table"
```

Comment:
Just a note that we made a distinction between "tables" and "catalogs". A 'catalog' is a list of sources (e.g. stars) with their attributes, it is encoded as a FITS table extension. A 'table' contains something else, e.g. a list of offsets for each detector, and we have not decided how it should be encoded. For example, a table with just four numbers can perhaps be better encoded as four FITS headers, because that makes it queryable through the archive. The term "catalog[ue] table" is never used in the DRLD in order to prevent confusion.

Reply:
My bad. This was probably autogenerated by some PyCharm AI voodoo and I failed to notice.